app/src/main/java/com/lassi/app/MainActivity.kt

```diff
@@ -54,6 +54,8 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
                     .setSupportedFileTypes("jpg", "jpeg", "png", "webp", "gif")
                     .enableFlip(true)
                     .enableRotate(true)
+                    .showSizeLabel(true)
+                    .setSupportedFileSize(10000)
```

Comment:
Here what is '10000'? mention file size unit in a comment. 
method name should be 'setMaxFileSize' instead of 'setSupportedFileSize'.